GOVERNANCE.md

```diff
@@ -0,0 +1,176 @@
+
+Consensus decisions can never override or go against the spirit of an earlier explicit vote.
+
+If any team member raises objections, the team members work together towards a solution that all involved can accept. This solution is again subject to lazy consensus.
+
+In case no consensus can be found, but a decision one way or the other must be made, any team member may call a formal majority vote.
+
+### Majority vote
+
+Majority votes must be called explicitly on the Metrictank [development mailing list][devlist]. It should reference any discussion leading up to this point.
+
+Votes may take the form of a single proposal, with the option to vote yes or no, or the form of multiple alternatives.
+
+A vote on a single proposal is considered successful if more vote in favor than against.
+
+If there are multiple alternatives, members may vote for one or more alternatives, or vote “no” to object to all alternatives. It is not possible to cast an “abstain” vote. A vote on multiple alternatives is considered decided in favor of one alternative if it has received the most votes in favor, and a vote from more than half of those voting. Should no alternative reach this quorum, another vote on a reduced number of options may be called separately.
+
+### Supermajority vote
+
+Supermajority votes must be called explicitly on the Metrictank [development mailing list][devlist]. It should reference any discussion leading up to this point.
+
+Votes may take the form of a single proposal, with the option to vote yes or no, or the form of multiple alternatives.
+
+A vote on a single proposal is considered successful if at least two thirds of those eligible to vote vote in favor.
+
+If there are multiple alternatives, members may vote for one or more alternatives, or vote “no” to object to all alternatives. A vote on multiple alternatives is considered decided in favor of one alternative if it has received the most votes in favor, and a vote from at least two thirds of those eligible to vote. Should no alternative reach this quorum, another vote on a reduced number of options may be called separately.
+
+## On / Offboarding
+
+The On / Offboarding section is informational and can be changed by [lazy consensus](#consensus) unless challenged. If no consensus can be reached, the matter may be resolved by [majority](#majority-vote) vote.
+
+### Onboarding
+
+Once a new member is approved they must enable 2 factor authentication for their Github account prior to starting the onboarding process. Then, the new member is:
+
+* Added to the list of team members, ideally by sending a PR of their own, or at least approving said PR
+* Announced in the *#metrictank-dev* channel in [slack][slack] and the Metrictank [development mailing list][devlist] by an existing team member
+* Added to the [Github project][gh] as a collaborator. Optionally, they are added to the community, and related organizations and repositories
+* Added to the private [team mailing list][team]
```

Comment:
does this allow them to see past threads or only emails since after they joined?

Reply:
I believe they have full access to the group at that point, so I think the answer is yes.